mentors/java/a.json

```diff
@@ -22,5 +22,13 @@
     "link_url": "https://www.dericbourg.net",
     "avatar_url": "http://www.dericbourg.net/images/profile.png",
     "bio": "Software engineer for 10 years now, mostly experienced on the JVM (Java and Scala) for web facing systems. Eager to learn, you'll probably find me trying new things on the other tracks!"
+  },
+    {
```

Comment:
The bracket should be aligned with the prior bracket.

Reply:
@kotp thanks, this is done.